src/main/java/com/gui/GUI.java

```diff
@@ -118,7 +103,16 @@ public void actionPerformed(ActionEvent e) {
             public void actionPerformed(ActionEvent e) {
                 if (EPUBFiles == null) return;
 
-                new Thread() {
+                Thread disableButtonsThread = new Thread() {
```

Comment:
The `open` and `disable` buttons do not disable when the generation processes beings. I decided to add another `Thread` object to deal with that. It's invoked right after the program makes sure the destination does not contain a duplicate file. What do you think?